distribution/docker/src/docker/iron_bank/download.json

```diff
@@ -1,7 +1,7 @@
 {
   "resources": [
     {
-      "url": "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${version}-linux-x86_64.tar.gz",
+      "url": "<artifact_path>/elasticsearch-${version}-linux-x86_64.tar.gz",
```

Comment:
What is this syntax? Does this value get replaced somewhere, and if so, how?

Reply:
It’s literally just a placeholder. Infra are planning a process to substitute the correct values during the unified release process.